src/couchapps/WMStatsAgent/views/agentRequests/map.js

```diff
@@ -0,0 +1,8 @@
+function(doc) {
+  if (doc.type === "agent_request" && doc.agent_url) {
+    // this condition check can be removed when there is no historical data is left. Not sure why this check is not working
+    //if (doc['_id'].startsWith(doc.agent_url)) {
+        emit(doc['_id'],  {'rev': doc['_rev']});
```

Comment:
Can't we simply emit `doc['_rev']` as value? instead of emitting a key/value dict.

Second question is regarding the database we store these docs on the agent side. What other types of documents are stored there?

Reply:
Yes we can, I think it is clearer and more flexible if we want to add some other values, but if you prefer we can change to just emit rev number.

Here is docs (view) stored in agents.
https://github.com/dmwm/WMCore/tree/master/src/couchapps/WMStatsAgent/views